demo/src/main/java/com/orange/ods/demo/ui/components/dialogs/DialogCustomizationState.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.demo.ui.components.dialogs
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+
+@Composable
+fun rememberDialogCustomizationState(
+    titleChecked: MutableState<Boolean> = rememberSaveable { mutableStateOf(false) },
+    dismissButtonChecked: MutableState<Boolean> = rememberSaveable { mutableStateOf(false) },
+) =
+    remember(titleChecked, dismissButtonChecked) {
+        DialogCustomizationState(titleChecked, dismissButtonChecked)
+    }
+
+class DialogCustomizationState(
```

Comment:
Not sure about this one, but from what I saw, it seems that states are named after their associated composable object. Here the composable is `ComponentDialogsContent` thus should we name the state `ComponentDialogsContentState` instead of `DialogCustomizationState` ?

Reply:
I renamed it and I think we should define naming conventions together cause I often ask myself questions about naming